eng/common/templates/jobs/source-build.yml

```diff
@@ -14,7 +14,7 @@ parameters:
   # This is the default platform provided by Arcade, intended for use by a managed-only repo.
   defaultManagedPlatform:
     name: 'Managed'
-    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8'
+    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8-20240104210103-4893224'
```

Comment:
Do we use the non-suffixed centos-stream8 container without this? Anything we check in to an `eng/common/*` file will get overwritten by the next arcade update - but my understanding was that the container we specify in `ci.yml` for SourceBuild should be the one that gets used, which would make this change unnecessary

Reply:
Ah, didn't know about it. I think `mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8` already contains what we need, I just used the tagged version and did a find and replace everywhere. If this gets overridden in the next arcade update, that's ok